DayrSuryan/DSEthiop13.xml

```diff
@@ -182,7 +182,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
-                    <seg type="script">Second half of the nineteenth-/early twentieth-century script of chant manuscripts.</seg>
+                    <seg type="script">Second half of the nineteenth-/early twentieth-century small script of
+                      manuscripts.</seg>
```

Comment:
no "chant" anymore? To me, it was clearer with it.

Reply:
ok, restituted.